src/compiler/passes/preprocess.js

```diff
@@ -0,0 +1,22 @@
+import util from '../utils/builder';
+
+export let createMethodHeaders = function(name, context) {
```

Comment:
agreed. this prolly doesn't make sense here. i could use your help figuring out what makes sense.